linux/README.txt

```diff
@@ -1,82 +0,0 @@
-======================= INIGRAL LINUX INTEGRATION README ======================
-
-This archive contains the sendcsv.sh script, which can be used in an automated
-way to integrate your student CSV data with Inigral via sFTP. This package
-contains a BASH script, sendcsv.sh, which should work on all modern versions
-of Linux and some other Unix-based OSes (Mac OS X etc) with the right packages.
-
-Version     - 1.0
-Date        - 2/1/2012
-
-=============================== Using sendcsv.sh ==============================
-
-You only need to set a few variables in the script for it to function. Then,
-you can run the script manually or schedule it to be run via the Control
-Panel's Scheduled Tasks panel.
-
- * Open sendcsv.sh with your favorite text editor
- 
- * Send your USERNAME, and PASSWORD
- 
-   * While passwords are great for testing, we suggest you use an SSH Key when
-     you decide to go to production. SSH keys are more secure and easy to use.
-	 Just follow the directions in this document, "Generating an SSH Key."
- 
- * By default, FILE will read from the first command-line parameter passed.
-   Alternatively, you can set FILE to the location of the CSV file you wish to 
-   upload. The file path is relative to the script's location, so if you had a 
-   file called your.csv in this directory, then it would work as configured. 
-   You can also use a full system file path (i.e. /path/to/file.csv).
- 
- * That's it! You can now run the script and it should upload the file to
-   Inigral. If you have any problems and wish to debug, feel free to read to
-   the end of the script. Lots of comments have been provided to assist.
-
-   Example: sendcsv.sh myfile.csv
-
-You can also integrate the sendcsv.sh into your existing processes. For an
-example of using sendcsv.sh, see sample_driver.sh. sample_driver.sh will
-upload an entire directory full of CSVs. To change the directory, open
-sample_driver.sh, and edit the DIRECTORY parameter.
-   
-============================ Generating an SSH Key ============================

```

Comment:
should these instructions be replaced somewhere else? are are we no longer going to use ssh?